### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by miadabdi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <title>Vite + React</title>
   </head>
   <body>
-    <div id="root"></div>
```
Comment (realtourani) [inline index.html:10]: Why did you change the id?

Comment (miadabdi) [reply index.html:10]: None of your business

**Review by realtourani — APPROVED** [review]
